src/tests/ftest/cart/SConscript

```diff
@@ -59,54 +61,81 @@ TEST_GROUP_NP_TESTS = ['test_group_np_srv.c', 'test_group_np_cli.c',
                        'no_pmix_group_version.c']
 FAULT_STATUS_TEST = 'fault_status.c'
 
+##############################
+# Create cart test library
+##############################
+def build_internal_library():
```

Comment:
rename maybe to build_testcommon_library() or some such; we might have multiple internal libraries going forward

Reply:
Need more info.  Would you like the name of the method change, or the name of the library changed, or both?

Reply:
Moved to cart/SConscript.  module and library name changed.